app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java

```diff
@@ -149,8 +149,8 @@
     private TextToSpeech textToSpeech;
     private String[] array;
     private String timenow;
-    private int remcount1 ;
-    private String remquery;
+    private int reminder_query ;
```

Comment:
You changed the variables to snake_case as well please change them to camelCase :) code unification is important :)